### PR TITLE
Main page

### PR DESCRIPTION
**Pull request by lbohdan**

### CODE REVIEW

File: js/components/filter/filter.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.filter {
+  background-color: rgb(255, 255, 255);
```
Comment (dosandk) [inline js/components/filter/filter.scss]: will be better use #color-hash style

File: js/components/filter/index.js [hunk 2]
```diff
@@ -1,23 +1,95 @@
...
+
+    return {
+      top: box.top + pageYOffset,
+      left: box.left + pageXOffset
+    };
   }
 
   render() {
+    const minYear = 1970;
+    const maxYear = new Date().getFullYear();
```
Comment (dosandk) [inline js/components/filter/index.js]: ```new Date().getFullYear()``` will be coll to move this to some component field or create a getter

File: js/components/filter/index.js [hunk 3]
```diff
@@ -1,23 +1,95 @@
...
+        />
+        <div className={styles['year-range']}>
+          <div>{minYear}</div>
+          <div>{maxYear}</div>
+        </div>
+        <SelectField
+          value={this.state.movieGenre}
+          onChange={this.handleGenreChange}
+        >
+          <MenuItem value={1} primaryText="Genres"/>
```
Comment (dosandk) [inline js/components/filter/index.js]: We should create config for all genres and construct ui-elements in loop

File: js/components/index/index.js [hunk 4]
```diff
@@ -5,7 +5,16 @@ import {List, ListItem} from 'material-ui/List';
 const Index = () => (
   <List>
     <Link to="/">
-      <ListItem primaryText="Home" />
+      <ListItem primaryText="Main page" />
```
Comment (dosandk) [inline js/components/index/index.js]: Links for menu will be better move to some config too

File: js/components/mainPage/mainPage.scss [hunk 5]
```diff
@@ -0,0 +1,13 @@
+.main-page-wrapper {
+  border: 2px solid red;
+  display: flex;
+  flex-flow: row wrap;
+
+  //& > * {
```
Comment (dosandk) [inline js/components/mainPage/mainPage.scss]: please, remove this comment lines

File: js/containers/carousel.js [hunk 6]
```diff
@@ -19,7 +19,7 @@ class Carousel extends Component {
   render() {
     return (
       <div>
-        {this.props.movies && this.props.movies.map(movie =>
+        {this.props.movies.map(movie =>
```
Comment (dosandk) [inline js/containers/carousel.js]: will be better always move 'map, filters, loops' from render to the some method

File: js/store/initial-store.js [hunk 7]
```diff
@@ -1,6 +1,13 @@
 const initialStore = {
-  // favoriteMovies: [],
```
Comment (dosandk) [inline js/store/initial-store.js:2]: please, remove commented lines

File: js/components/filter/index.js (hunk 2 above)
Comment (lbohdan) [reply js/components/filter/index.js]: could you please give an example with getter?

**Review by dosandk — APPROVED** [review]

**Pull request merged**